boards/arm/dragonclaw/dragonclaw.dts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <st/f4/stm32f412Xg.dtsi>
+#include <st/f4/stm32f412z(e-g)tx-pinctrl.dtsi>
+
+/ {
+	model = "Google Dragonclaw development board";
+	compatible = "google,dragonclaw-fpmcu", "st,stm32f412";
```

Comment:
I think this should just be `compatible = "google,dragonclaw-fpmcu"`, not that it's used for anything, but it should refer to the specific board afaik.